ozaria/site/components/interactive/PageInteractive/insertCode/index.vue

```diff
@@ -75,32 +74,57 @@
 
         let selectedAnswerLine = ''
```

Comment:
This method can be simplified now that we have lineToReplace.  We no longer need to split the sample code into two parts.  We can break it into an array once on component load and then replace the line configured in `lineToReplace` when we have a selected answer.

Reply:
I agree 100%.